modules/nf-core/kallistobustools/count/main.nf

```diff
@@ -16,9 +16,11 @@ process KALLISTOBUSTOOLS_COUNT {
     val   technology
 
     output:
-    tuple val(meta), path ("*.count"), emit: count
-    path "versions.yml"              , emit: versions
-    path "*.count/*/*.mtx"           , emit: matrix //Ensure that kallisto finished and produced outputs
+    tuple val(meta), path ("*.count")                  , emit: count
+    tuple val(meta), path ("*.count/counts_unfiltered"), emit: raw_counts                       // TODO: Add to nf-coew/modules before merging PR
+    tuple val(meta), path ("*.count/counts_filtered")  , emit: filtered_counts, optional: true  // TODO: Add to nf-coew/modules before merging PR
```

Comment:
Here I am aware that this modification must go to nf-core/modules and not here, thus I added the TODO so this removed once testing is done.